### PR TITLE
Remove `isCernerLive` in selector

### PR DESCRIPTION
**Pull request by kelsonic**
## Description
This PR removes the `isCernerLive` logic in the `selectPatientFacilities` selector.

## Testing done
N/A

## Screenshots
N/A

## Acceptance criteria
- [x] Remove `isCernerLive` from `selectPatientFacilities`.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by ncksllvn — APPROVED** [review]

File: src/platform/user/selectors.js [hunk 1]
```diff
@@ -12,11 +13,27 @@ export const isLOA1 = state => selectProfile(state).loa.current === 1;
...
+    const hasCernerFacilityID = CERNER_FACILITY_IDS.includes(facilityId);
+
+    // Derive if the feature toggle is on.
+    const showNewScheduleViewAppointmentsPage =
+      state?.featureToggles?.[
+        featureFlagNames.showNewScheduleViewAppointmentsPage
+      ];
+
+    // Derive if they are a 200CRNR Cerner patient.
+    const isCernerPatient = state?.user?.profile?.isCernerPatient;
```
Comment (jbalboni) [inline src/platform/user/selectors.js:27]: You could use the selectProfile selector here. And the feature toggles one above. It reduces the risk that someone tweaks the redux state and breaks code not using the expected selectors. Not a big thing, though

**Review by jbalboni — APPROVED** [review]
LGTM

File: src/platform/user/selectors.js (hunk 1 above)
Comment (kelsonic) [reply src/platform/user/selectors.js:27]: Updated! Thanks! 🙂

Comment (kelsonic) [reply src/platform/user/selectors.js:27]: I was worried we were using reselect everywhere for all these selectors, which can cause some serious perf issues when misused, but now just seeing that these are just helper funcs I can see the value in them 👍

**Review by cvalarida — APPROVED** [review]

**Pull request merged**